drv/cosmo-hf/src/main.rs

```diff
@@ -291,4 +302,13 @@ impl ServerImpl {
     }
 }
 
+/// Failure function, running an Idol response loop that always returns an error
+fn fail(err: drv_hf_api::HfError) {
+    let mut buffer = [0; hf::idl::INCOMING_SIZE];
+    let mut server = hf::FailServer { err };
+    loop {
+        idol_runtime::dispatch(&mut buffer, &mut server);
+    }
+}
```

Comment:
This is a nice approach to having a driver go into "failure mode" when hardware can't be initialized, I wonder if we might want to do something similar in e.g. `gimlet_seq`, here:
https://github.com/oxidecomputer/hubris/blob/6990578c7f09bfa75784e6cfa923bbd1dace649f/drv/gimlet-seq-server/src/main.rs#L159-L176
rather than just waiting for a non-existent notification?

Obviously not something to do in this PR, but could be nice to do later.